website/docs/extensions/write/onboarding-workflow.md

```diff
@@ -0,0 +1,531 @@
+
+A practical example of progressing the user to the next step after the command finishes its execution is
+
+```json
+"commands": [
+    {
+    "command": "podman.onboarding.checkPodmanInstalled",
+    "title": "Podman: Check podman installation"
+    },
+],
+"onboarding": {
+    "title": "Podman Setup",
+    "steps": [
+    {
+        "id": "checkPodmanInstalled",
+        "title": "Checking for Podman installation",
+        "command": "podman.onboarding.checkPodmanInstalled",
+        "completionEvents": [
+            "onCommand:podman.onboarding.checkPodmanInstalled"
+        ]
+    },
+    ...
+    ],
+    ...
+}
+```
+
+When the `checkPodmanInstalled` step becomes active, the command `podman.onboarding.checkPodmanInstalled` is invoked. Upon completion of its execution, the step is considered complete, and the user is then moved to the next one.
+
+Here's another example, this time using a context value:
+
+```json
+"commands": [
+    {
+    "command": "podman.onboarding.checkPodmanInstalled",
+    "title": "Podman: Check podman installation"
+    },
+],
+"onboarding": {
+    "title": "Podman Setup",
+    "steps": [
+    {
+        "id": "checkPodmanInstalled",
+        "title": "Checking for Podman installation",
+        "command": "podman.onboarding.checkPodmanInstalled",
+        "completionEvents": [
+            "onboardingContext:podmanVersion == 4.7.2"
+        ]
+    },
+    ...
+    ],
+    ...
+}
+```
+
+When the `checkPodmanInstalled` step becomes active, the command `podman.onboarding.checkPodmanInstalled` is invoked. As soon as the context value `podmanVersion` equals `4.7.2`, the step is marked as completed, and the user is moved to the next one.
+
+You might wonder: who or what sets the context value? If you use a custom context value, it should be your extension's job to set it. Following the example above, we could set the context value during the execution of `podman.onboarding.checkPodmanInstalled` like
+
+```
+extensionApi.commands.registerCommand(
+    'podman.onboarding.checkPodmanInstalled',
+    async () => {
+      // do something
+      ...
+      // set podmanVersion context value so we can mark the step as complete
+      extensionApi.context.setValue('podmanVersion', '4.7.2', 'onboarding');
+    }
+)
+```
+
+After updating the context, the UI is refreshed, and Podman Desktop moves the user to the new step.
+
+#### Content
+
+The **content** property is an array of arrays where each item in the parent array defines a row, and each item in the child arrays defines a cell.
+
+```
+content = [
+    ["cell", "cell"], //row
+    ["cell", "cell", "cell"], //row
+]
+```
+
+The JSON schema for a content cell entry is
+
+```json
+"type": "object",
+"properties": {
+    "value": {
+        "type": "string"
+    },
+    "highlight": {
+        "type": "boolean"
+    },
+    "when": {
+        "type": "string"
+    }
+},
+"required": ["value"]
+```
+
+With **value** being the only mandatory field.
```

Comment:
'With **value** being' -> '**value** is', and maybe make these two sentences a single paragraph.